deploy/contents/k8s/efs-csi/resources/efs-pvc.yaml

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: efs-claim
```

Comment:
rename this pvc to `cp-system-pvc` to be compatible with cp-aws-native resources definition